backend/prompt_studio/prompt_studio_output_manager/views.py

```diff
@@ -49,3 +53,56 @@ def get_queryset(self) -> Optional[QuerySet]:
             queryset = PromptStudioOutputManager.objects.filter(**filter_args)
 
         return queryset
+
+    def get_output_for_tool_default(self, request: HttpRequest) -> Response:
+        # Get the tool_id from request parameters
+        # Get the tool_id from request parameters
+        tool_id = request.GET.get("tool_id")
+        document_manager_id = request.GET.get("document_manager")
+
+        if not tool_id:
+            return Response(
+                {"error": "tool_id parameter is required"},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
```

Comment:
@jagadeeswaran-zipstack I suggest using a serializer instead to validate this in order to be consistent with our error messages. If you think it might be an overkill here, raise an `APIException` with the expected message and status code as 400 instead. Refrain from framing your own error messages in this way, please propagate this change to all other places where this practice is followed

Reply:
Got it